### PR TITLE
S2task1 add to database

### PR DESCRIPTION
**Pull request by Freschwamm**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -29,7 +27,15 @@
  <div>
  <?php echo pokemonCardContainer($allPokemon) ?>
```
Comment (MrLukey) [inline index.php]: Indentation

File: index.php [hunk 2]
```diff
@@ -29,7 +27,15 @@
...
  </div>
-
+ <div>
+     <form action="validation.php" method="post">
+         <input type="text" placeholder="Input pokemon name" name="pokemonName" required>
+         <input type="text" placeholder="Input card cost" name="pokemonCost" required>
+         <input type="text" placeholder="Input pokemon card image link" name="pokemonImage" required>
+         <input type="text" placeholder="Input card rarity" name="pokemonRarity" required>
```
Comment (MrLukey) [inline index.php]: WAVE - Inputs have no labels

File: index.php [hunk 3]
```diff
@@ -29,7 +27,15 @@
...
+         <input type="submit" name="submit">
```
Comment (MrLukey) [inline index.php]: WAVE - submit button needs a value

File: validation.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?php
+$db = new PDO('mysql:host=db; dbname=pokemon-cards', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT * FROM `pokemon-card`;');
+
+$query->execute();
+
+$allPokemon = $query->fetchAll();
```
Comment (MrLukey) [inline validation.php]: dont need to pull data into this page as none is used

**Review by MrLukey — CHANGES_REQUESTED** [review]

File: validation.php (hunk 4 above)
Comment (Freschwamm) [reply validation.php]: Done

File: index.php (hunk 1 above)
Comment (Freschwamm) [reply index.php]: done

**Review by MrLukey — APPROVED** [review]
All gravy

**Pull request merged**